supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorWrapper.scala

```diff
@@ -153,4 +153,5 @@ object HtCondorRuntimeKeys {
   val Disk = "request_disk"
   val LogXml = "log_xml"
   val LeaveInQueue = "leave_in_queue"
+  val InitialWorkingDir = "Iwd"
```

Comment:
Oh, so this is nothing to do with runtime attributes in the WDL file? I just saw "runtime" in the name... Mea Culpa! :)